src/components/Options/TestDetailsOptions.js

```diff
@@ -0,0 +1,47 @@
+const TestTypeOptions = [
```

Comment:
Choose same naming scheme for all variables (see above comment)

Reply:
Done All now follow same naming scheme